Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -1263,6 +1266,92 @@ private static void OnIsValidPropertyChanged(DependencyObject d, DependencyPrope
             }
         }
 
+        // Field used for accessing the ItemsSource though the interface ISupportIncrementalLoading
+        private ISupportIncrementalLoading _incrementalItemsSource;
+
+        private bool _loadingItems;
+
+        // Method that loads additional data once the user has scrolled to a certain threashold
+        private async void LoadMoreDataFromIncrementalItemsSource()
+        {
+            if (IncrementalLoadingTrigger == IncrementalLoadingTrigger.Edge && _incrementalItemsSource != null && _incrementalItemsSource.HasMoreItems && !_loadingItems)
+            {
+                var bottomScrollOffHeight = Math.Max(0, EdgedRowsHeightCalculated - CellsHeight - VerticalOffset);
+
+                var rowsPresenterAvailableHeight = this._rowsPresenterAvailableSize.HasValue
+                    ? _rowsPresenterAvailableSize.Value.Height
+                    : _vScrollBar.ViewportSize;
+
+                if ((IncrementalLoadingThreshold * rowsPresenterAvailableHeight) >= bottomScrollOffHeight)
+                {
+                    _loadingItems = true;
+
+                    var numberOfRowsToLoad = (uint)(DataFetchSize * rowsPresenterAvailableHeight / this.RowHeightEstimate);
+
+                    await _incrementalItemsSource.LoadMoreItemsAsync(numberOfRowsToLoad);
+
+                    _loadingItems = false;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Identifies the IncrementalLoadingThreshold dependency property
+        /// </summary>
+        public static readonly DependencyProperty IncrementalLoadingThresholdProperty =
+            DependencyProperty.Register(
+                "IncrementalLoadingThreshold",
```

Comment:
use nameof

Reply:
Looking at all other dependency properties in DataGrid.cs, it seems like the convention is to spell out the name rather than using nameof. As such, I'd prefer to keep it the way it is to avoid different. Thoughts?

Reply:
Agree, we should use `nameof` wherever possible, please update the other occurrences.